Dockerfile

```diff
@@ -1,10 +1,12 @@
 # Build the manager binary
-FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:1.14 as builder
+FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:1.14-alpine as builder
 
 WORKDIR /workspace
 # Copy the Go Modules manifests
 COPY go.mod go.mod
 COPY go.sum go.sum
+# Set GOPROXY if downloading slowly
+RUN go env -w GOPROXY=https://goproxy.cn,https://goproxy.io,direct
```

Comment:
will it slow down the speed outside of China?

Reply:
`GOPROXY` is really frustrating, so I will remove it first

Reply:
Done